src/test/regress/expected/multi_alter_table_add_constraints.out

```diff
@@ -0,0 +1,501 @@
+INSERT INTO unique_test_table VALUES(1, 'Mehmet');
+ERROR:  duplicate key value violates unique constraint "unn_id_name_1400035"
+DETAIL:  Key (id, name)=(1, Mehmet) already exists.
+CONTEXT:  while executing command on localhost:57638
+DROP TABLE unique_test_table;
+-- Check "UNIQUE CONSTRAINT" with reference table
+CREATE TABLE unique_test_table_ref(id int, name varchar(20));
+SELECT create_reference_table('unique_test_table_ref');
+ create_reference_table 
+------------------------
+ 
+(1 row)
+
+-- We can add unique constraint on any column with reference tables
+ALTER TABLE unique_test_table_ref ADD CONSTRAINT unn_name UNIQUE(name);
+ALTER TABLE unique_test_table_ref ADD CONSTRAINT unn_id UNIQUE(id);
+-- Error out. Since the table can not have two rows with the same id. 
+INSERT INTO unique_test_table_ref VALUES(1, 'Ahmet');
+INSERT INTO unique_test_table_ref VALUES(1, 'Mehmet');
+ERROR:  duplicate key value violates unique constraint "unn_id_1400066"
+DETAIL:  Key (id)=(1) already exists.
+CONTEXT:  while executing command on localhost:57637
+-- We can add unique constraint with multiple columns 
+ALTER TABLE unique_test_table_ref DROP CONSTRAINT unn_id;
+ALTER TABLE unique_test_table_ref ADD CONSTRAINT unn_id_name UNIQUE(id,name);
+INSERT INTO unique_test_table_ref VALUES(1, 'Mehmet');
+DROP TABLE unique_test_table_ref;
+-- Check "UNIQUE CONSTRAINT" with append table
+CREATE TABLE unique_test_table_append(id int, name varchar(20));
+SELECT create_distributed_table('unique_test_table_append', 'id', 'append');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+-- Cannot add unique constraint on append table
+ALTER TABLE unique_test_table_append ADD CONSTRAINT unn_name UNIQUE(name);
+ERROR:  cannot add UNIQUE, PRIMARY KEY or EXCLUDE constraint on "unique_test_table_append" 
+DETAIL:  UNIQUE constraints, EXCLUDE constraints, and PRIMARY KEYs on append-partitioned tables cannot be enforced.
+HINT:  Consider using hash partitioning.
+ALTER TABLE unique_test_table_append ADD CONSTRAINT unn_id UNIQUE(id);
+ERROR:  cannot add UNIQUE, PRIMARY KEY or EXCLUDE constraint on "unique_test_table_append" 
+DETAIL:  UNIQUE constraints, EXCLUDE constraints, and PRIMARY KEYs on append-partitioned tables cannot be enforced.
+HINT:  Consider using hash partitioning.
+COPY unique_test_table_append FROM STDIN DELIMITER AS ',';
+DROP TABLE unique_test_table_append;
+-- Check "CHECK CONSTRAINT"
+CREATE TABLE products (
+    product_no integer,
+    name text,
+    price numeric,
+    discounted_price numeric
+);
+SELECT create_distributed_table('products', 'product_no');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+-- Can add column and table constraints
+ALTER TABLE products ADD CONSTRAINT p_check CHECK(price > 0);
+ALTER TABLE products ADD CONSTRAINT p_multi_check CHECK(price > discounted_price);
+-- Error out from the related shard. Table can not have row conflicting with p_multi_check
```

Comment:
"Related shard" term does not feel right here. Lets make this (and others) something similar to;

"This query will error out because insterting row's price is less than 0.